src/Translate/TranslateClient.php

```diff
@@ -92,20 +90,22 @@ class TranslateClient
      */
     public function __construct(array $config = [])
```

Comment:
Few things here:

1) Can we remove the `@throws` statement?

2) Now that we support oauth we should also document the rest of the configuration parameters such as `authHttpHandler`, `keyFile`, `scopes` etc.

3) We need to call `$this->configureAuthentication()` if an api key isn't provided